### ISSUE TITLE
Fails with Java 9 Early Access

### ISSUE THREAD

**Issue by jed-frey** [marytts/marytts#665]
The version string in Java 9's Early Access is "9-ea", causing the EnvironmentCheck to fail.

Stack:

```
marytts-5.2/bin$ ./marytts-server 
java version "9-ea"
Java(TM) SE Runtime Environment (build 9-ea+140)
Java HotSpot(TM) 64-Bit Server VM (build 9-ea+140, mixed mode)

MARY server 5.2 starting as a HTTP server...Exception in thread "main" java.lang.NumberFormatException: For input string: "9-e"
	at jdk.internal.math.FloatingDecimal.readJavaFormatString(java.base@9-ea/FloatingDecimal.java:2054)
	at jdk.internal.math.FloatingDecimal.parseFloat(java.base@9-ea/FloatingDecimal.java:122)
	at java.lang.Float.parseFloat(java.base@9-ea/Float.java:451)
	at marytts.server.EnvironmentChecks.check(EnvironmentChecks.java:46)
	at marytts.server.Mary.startup(Mary.java:286)
	at marytts.server.Mary.startup(Mary.java:204)
	at marytts.server.Mary.main(Mary.java:513)
```

```
marytts-5.2/bin$ java -version
java version "9-ea"
Java(TM) SE Runtime Environment (build 9-ea+140)
Java HotSpot(TM) 64-Bit Server VM (build 9-ea+140, mixed mode)
```

**Comment by seblemaguer** [comment]
Hello,

thanks for the report, for now we support java 7/8. We will have a look at it as soon as possible but if you need an urgent solution I suggest to switch back to these jdk.

**Comment by seblemaguer** [comment]
solved by #923
